project-base/assets/js/frontend/components/hoverIntent.js

```diff
@@ -1,7 +1,8 @@
 import Responsive from '../utils/responsive';
 import HoverIntentSetting from './HoverIntentSetting';
 import Register from 'framework/common/utils/register';
-import 'jquery-hoverintent';
+import $ from 'jquery';
```

Comment:
I noticed you used different way to get jQuery in previous files.
`const $ = window.jQuery || global.jQuery || jQuery;` 

Is any difference between these two? Can it be unified?

Reply:
It is my nightmare. When we use hack for using jsformvalidator out of webpack, we have to set jquery as global variable and some jquery plugins have problems with it. And we have our own plugins (To make things worse). I belive that we can unite this when we use form validator in webpack.

But in admin/index.js I removed global jquery. It was my forgot parth of code for testing.